test/embroider/update-manifest-from-html.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
```

Comment:
You mentioned a scenario that https://github.com/ember-fastboot/ember-cli-fastboot/pull/727 didn't handle, but these tests appear to be the same as https://github.com/ember-fastboot/ember-cli-fastboot/pull/727 could you add the missing test, or leave a note as to the test change you added that covers that scenario?